modules/Hoot/managers/api.js

```diff
@@ -1260,9 +1260,7 @@ export default class API {
         const params = {
             url: `${ this.translationUrl }schema`,
             method: 'GET',
-            params: {
-                ...data
-            }
+            params: data
```

Comment:
I kept getting lint errors, and it turns out that the `{...obj}` syntax is an es6 way to make a clone of an object.
I don't think we are doing anything except making a request with these param objects such that we are not creating vulnerability to the scenario where you create a bug because you update a property on an object you didn't know was used elsewhere.

So, I `just` removed the spread stuff... If we want to be careful I guess we could just do Object.assign({},d)`.